include/xen/be/XenGnttab.hpp

```diff
@@ -213,10 +214,20 @@ class XenGnttabDmaBufferImporter
 			delete;
 	~XenGnttabDmaBufferImporter();
 
+#ifdef GNTTAB_HAS_DMABUF_OFFSET
```

Comment:
I'd prefer to remove `#ifdef`s  in the header. It is better to have stable interface for users. Users don't want to make conditional compilation only because you are linking against another lib with some feature missing,

You can safely return offset `0` if `GNTTAB_HAS_DMABUF_OFFSET` is not defined.

Reply:
makes sense

Reply:
fixed